lib/rubocop/cop/style/option_hash.rb

```diff
@@ -50,6 +53,11 @@ def on_args(node)
         def supports_keyword_arguments?
           RUBY_VERSION >= '2.0.0'
         end
+
+        def name_in_suspicious_param_names?(arg_name)
+          cop_config.key?('SuspiciousParamNames') &&
+            cop_config['SuspiciousParamNames'].include?(arg_name.to_s)
```

Comment:
```ruby
# for users who opted in and have configuration:
# Style/OptionHash:
#   enabled: true
cop_config.key?('SuspiciousParamNames') #=> false
```
I *think* they'll always return false from this method because their configuration won't have this key, and the cop will silently ignore option hashes, even though they thought they opted into it. It won't be an error, but if that analysis is right, it won't behave how they expected... does it make sense?